library/src/main/java/xyz/klinker/android/article/ArticleIntent.java

```diff
@@ -616,4 +636,21 @@ public static boolean shouldAlwaysUseBrowserUI(Intent intent) {
         return intent.getBooleanExtra(EXTRA_USER_OPT_OUT_FROM_CUSTOM_TABS, false)
                 && (intent.getFlags() & Intent.FLAG_ACTIVITY_NEW_TASK) != 0;
     }
+
+    /**
+     * Converts the ArticleIntent integer theme to a DragDismiss theme
+     *
+     * @param themeInt One of {@link #THEME_LIGHT}, {@link #THEME_DARK}, {@link #THEME_BLACK} or
+     *                  {@link #THEME_AUTO}.
+     * @return the corresponding DragDismiss theme
+     */
+    static DragDismissIntentBuilder.Theme convertIntToTheme(int themeInt) {
+        switch (themeInt) {
+            case THEME_LIGHT: return DragDismissIntentBuilder.Theme.LIGHT;
```

Comment:
Fix the formatting on these. Either bring the default up to the same line or move these down to the next line.